syntax = "proto3";
package system_map;





/*
CollectSystem请求
*/
message CollectSystemRequest {
    /*
    应用系统ID
    */
    string systemId = 1 ;
    /*
    操作类型, [favorite 收藏] [unfavorite 取消收藏]
    */
    string operation = 2 ;
}

/*
CollectSystem返回
*/
message CollectSystemResponse {
    /*
    收藏 id
    */
    string instanceId = 1 ;
}

/*
CollectSystemApi返回
*/
message CollectSystemResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.CollectSystemResponse data = 4 ;
}

